`fly services list` failing on Machines apps

fly v0.0.457 darwin/arm64 Commit: f58a2e68 BuildDate: 2023-02-15T01:31:03Z

```
$ fly services list -a appname         
Oops, something went wrong! Could you try that again?
```

Error from @lillianberryfly's local build from master:
```
panic: interface conversion: interface {} is nil, not *flaps.Client
```

--- a/internal/command/services/machines.go
+++ b/internal/command/services/machines.go
@@ -61,7 +61,7 @@ func showMachineServiceInfo(ctx context.Context, app *api.AppInfo) error {
 
 			fields := []string{
 				strings.ToUpper(protocol),
-				fmt.Sprintf("%d => %d [%s]", port.Port, service.InternalPort, strings.Join(handlers, ",")),
+				fmt.Sprintf("%d => %d [%s]", *port.Port, service.InternalPort, strings.Join(handlers, ",")),
 				strings.Title(fmt.Sprint(port.ForceHttps)),
 			}
 			services = append(services, fields)

